### PR TITLE
Add support for multiple entry files

### PR DESCRIPTION
**Pull request by facelesspanda**
Adds support for multiple i18n files.

This does not change any current behaviour. It only adds the possibility to specify a different module to look into if so desired.

Allows users to have several i18n files and use them as they see fit.
A use case could be thematic splitting of different phrases into different files (eg. buttons, notifications, dates)

~~I'm using a `Pexp_construct` to match the lexing specificities of module names, but I may have missed something. I wouldn't like to have to write `[%i18n ((module MyI18n), the_entry)]` because of the verbosity...~~

**EDIT**: I changed the rewriter's behavior to match against the given module path.
Basically if it's `[%i18n ident]` or `[%i18n S.ident]` it uses the default module name (ie no change).
If it's `[%i18n MODULE.ident]` or `[%i18n MODULE.S.ident]` it will use `MODULE` as the module name.
It will consider any path longer than that an error. `[%i18n MODULEA.MODULEB.ident]` is also an error.

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -18,13 +18,26 @@ usage: ocsigen-i18n-generator [options] [< input] [> output]
...
+(basename of the output file)
 
 This will call the right function turning
-`[%i18n foo]` into `MODULE_NAME.foo ()`,
-`[%i18n S.bar ~x:"foo"]` into `MODULE_NAME.S.bar ~x:"foo" ()]` and so on...
+`[%i18n foo]` into `DEFAULT.foo ()`,
+`[%i18n S.bar ~x:"foo"]` into `DEFAULT.S.bar ~x:"foo" ()]` and so on...
+
+To specify a different module from the default one, you can do so using
+a expressions such as these:
```
Comment (remyzorg) [inline README.MD:38]: expressions

File: README.MD [hunk 2]
```diff
@@ -18,13 +18,26 @@ usage: ocsigen-i18n-generator [options] [< input] [> output]
...
+ - `[%i18n MyI18n foo]` will become `MyI18n.foo ()`
```
Comment (remyzorg) [inline README.MD:39]: Why not `MyI18n.foo` ?

Comment (facelesspanda) [reply README.MD:39]: because of the `S.foo` possibility. I guess we could check the length of the dot chain and extract the first one if it's more than 2, but I really don't know how robust this is.

Comment (remyzorg) [reply README.MD:39]: Ok.

**Comment by facelesspanda** [comment]
@remyzorg @balat is this okay with you now?
